src/model/StateStorage.py

```diff
@@ -237,6 +247,9 @@ def __Load(fileName):
         # puts elements in the file in a dictionary
         dict = json.load(file)
         obj = __setFields(dict)
+        os.chdir('..')
+        os.chdir('..')
+        os.chdir('..')
         return obj
     except FileNotFoundError:
```

Comment:
This is where that file crash error is happening. Get this fixed to CD on a fail as well (probably want a finally clause for cleanup regardless of if exception is thrown of not)